frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -125,6 +128,75 @@ export default function AddUser(): JSX.Element {
     });
   }
 
+  async function uploadUsersCsv(csvFile: File | undefined): Promise<void> {
+    if (csvFile) {
+      const csv = await csvFile.text();
+      const json = await csvtojson().fromString(csv);
+      const allNetIds = allMembers.map((m) => m.netid);
+      const members: IdolMember[] = [];
+      if (json[0].email) {
+        json.forEach((m) => {
+          const currMember = allMembers.find((mem) => mem.netid === getNetIDFromEmail(m.email));
+          if (currMember) {
+            members.push({
```

Comment:
I like to avoid code duplication if possible, but not while sacrificing readability. But I think we can do a little better here. Thoughts on this?

Create a generic "EMPTY_MEMBER" object and place it up top as a constant. Line 84 in this file can use this object to set the state.

```
const EMPTY_MEMBER = {
        firstName: '',
        lastName: '',
        pronouns: '',
        email: '',
        role: '' as Role,
        graduation: '',
        major: '',
        doubleMajor: '',
        minor: '',
        website: '',
        linkedin: '',
        github: '',
        hometown: '',
        about: '',
        subteams: [],
        formerSubteams: []
}
```

Then we use this same object to loop through the keys of the object:

```
const member = {};
Object.keys(EMPTY_MEMBER).forEach((field) => { 
 member[field] = m[field] ?? currMember[field]
}
members.push(member)
```
And then down below we can do 

```
const member = {...EMPTY_MEMBER};
Object.keys(EMPTY_MEMBER).forEach((field) => { 
 if(m[field])
   member[field] = m[field]
}
members.push(member)
```